fv/containers/containers.go

```diff
@@ -66,7 +66,7 @@ func Run(namePrefix string, args ...string) (c *Container) {
 
 	// Prep command to run the container.
 	log.WithField("container", c).Info("About to run container")
-	runArgs := append([]string{"run", "--rm", "--name", c.Name, "--hostname", c.Name}, args...)
+	runArgs := append([]string{"run", "--name", c.Name, "--hostname", c.Name}, args...)
```

Comment:
The `--rm` was very-much intentional here.  It basically makes cleanup automatic if the test is `Ctrl-C`ed, which is doubly important because Felix's FVs can be run in parallel making automated cleanup tricky.  The cleanup takes advantage of the fact that we leave the docker CLI process running and it automatically gets killed if our process dies.

Reply:
@ozdanborne and I discussed yesterday.  He has a requirement for being able to stop and then restart a container.  @ozdanborne Is it the case that a container with `--rm` will be destroyed completely when it is stopped?  If so, then clearly the containers that you want to restart can't have `--rm`.  However, as @fasaxc says, we do still want `--rm` for most containers - or else some equally robust cleanup solution - and in any case you'll also need a robust cleanup solution for the containers that you want to stop and restart.

Reply:
Why do we need to clean up stopped containers?

Reply:
As discussed offline, cleanup was one of the problems we were trying to solve here for the felix FVs but happy for this to be extended with a new use-case as long as we don't lose the cleanup that we wanted for felix FVs.